widget/embedded/src/pages/Home.tsx

```diff
@@ -137,8 +138,12 @@ export function Home() {
 
   const onChangeSlippage = (slippage: number | null) => {
     if (slippage) {
-      setSlippage(slippage);
-      setCustomSlippage(null);
+      if (SLIPPAGES.includes(slippage)) {
+        setSlippage(slippage);
+        setCustomSlippage(null);
+        return;
+      }
+      setCustomSlippage(slippage);
     }
   };
```

Comment:
You can do it also for the first condition:
```suggestion
      return;
      }
      if (SLIPPAGES.includes(slippage)) {
        setSlippage(slippage);
        setCustomSlippage(null);
        return;
      }
      setCustomSlippage(slippage);
  };
```